### DISCUSSION TITLE
Should we use MiKTeX or TeX Live on Windows?

### DISCUSSION THREAD

**Discussion by CoolOppo** [3b1b/manim#1550] [Q&A] [answered]
The [README](https://github.com/3b1b/manim/blob/master/README.md#directly-windows) and [installation page](https://3b1b.github.io/manim/getting_started/installation.html#directly-windows) both have different recommendations as to which TeX distribution to use on Windows. It seems to me that one has to be the newer recommendation, but I'm not really sure which to go with.

**Comment by TonyCrane** [comment, marked as answer]
Both are ok. You only need to make sure that `latex` command work properly and all the packages used in `tex_template.tex` are installed.
As for me, TeXLive-full work well.

**Reply by CoolOppo** [reply]
That's good to hear at least. Just wondering if there's a reason there are two different recommendations I guess. I would like to be using the "best" one. I might actually go with MiKTeX because it supports the community version too though.

**Reply by TonyCrane** [reply]
TeXLive also supports the community version.
